client/state/ui/reducer.js

```diff
@@ -76,14 +77,23 @@ export function isLoading( state = false, action ) {
 	return state;
 }
 
+export function guidesTour( state = {}, action ) {
+	switch ( action.type ) {
+		case SHOW_GUIDESTOUR:
+			return { show: action.show, tour: action.tour, stepName: action.stepName, siteId: action.siteId };
```

Comment:
I like that `show` is a short word, but it's also a very ambiguous one (adjective, verb, noun…). Wondering whether we should have something more like `shouldShow`, `isShowing`, whatever.